test/alternator/test_gsi.py

```diff
@@ -4,6 +4,10 @@
 
 # Tests of GSI (Global Secondary Indexes)
 #
+# All the tests in this file create a GSI together with the base table - the
+# separate file test_gsi_updatetable.py has tests for the ability to use
+# UpdateTable to add or remove GSIs on existing tables.
+#
```

Comment:
> This patch adds many more tests for various corner cases of
> this feature - tests developed in parallel with actually implementing
> that feature.
> 
> The source file test/alternator/test_gsi.py has already grown pretty
> large, so this patch moves all the tests related to using UpdateTable
> to add or delete a GSI - both the existing tests and the new ones -
> to a separate file: test_gsi_updatetable.py,

Please separate the code movement and addition of new tests into two commits.

Reply:
I can do this, but it won't be trivial, as I actually made some changes in comments and perhaps (I don't remember) other small changes in the code after the movement. 

From the **deletion** side you can see exactly what I moved (which got deleted from the old file), but you're right that you can't see exactly what I changed after the move.